src/command.rs

```diff
@@ -1,12 +1,13 @@
 use std::fs;
 use std::io::Read;
+use std::process;
 use std::process::Stdio;
```

Comment:
Nit: import the module with `self` instead.

```suggestion
use std::process::{self, Stdio};
```